scripts/reimage_fog.py

```diff
@@ -0,0 +1,22 @@
+import docopt
+
+from teuthology.misc import reimage_fog
+import sys
+
+doc = """
+usage: teuthology-reimage-fog -h
+       teuthology-reimage-fog --nodes node1,node2 --os-type distro --os-version version
```

Comment:
It is more usable to provide node names without a key, so bash expansion can be used, like in teuthology-lock:
```
teuthology-reimage-fog --os-type distro --os-version X.Y node{1,2}
```